### PR TITLE
[RW-9042][risk=no] Fix spelling error in demographics survey

### PR DESCRIPTION
**Pull request by dolbeew**
Fix spelling of Ethnicity option in the demographics survey from 'Columbian' to 'Colombian'

@jmthibault79 tagging you for review since you were the top suggestion by github. Mainly want to make sure I'm not breaking anything by changing the spelling of the enum property.

### CODE REVIEW

File: api/src/main/resources/workbench-api.yaml [hunk 1]
```diff
@@ -9957,7 +9957,7 @@ definitions:
       - BLACK_OTHER           # Black / None of these fully describe me, and I want to specify
 
       - HISPANIC              # Hispanic, Latino, or Spanish descent
-      - HISPANIC_COLUMBIAN
+      - HISPANIC_COLOMBIAN
```
[the remark this answers is not in the document]
Comment (dolbeew) [reply api/src/main/resources/workbench-api.yaml:9960]: Ok, thanks. I'll give it a shot and if it ever looks like it won't get done, will just push the UI change.

**Comment by dolbeew** [comment]
This is working locally for me now. Before adding the changelog to rename the column, submitting the survey locally failed with a 500. After renaming the column and running `./project.rb run-local-all-migrations`, I'm able to submit the survey with no errors.

**Review by jmthibault79 — APPROVED** [review]
Looks good - works for me as well.

Would you mind adding some comments to the SQL statements to clarify how the enums are being changed and make the add -> convert -> drop sequence more explicit?

**Comment by dolbeew** [comment]
Good call, done.

**Pull request merged**